docs/.vuepress/public/_redirects

```diff
@@ -53,20 +53,100 @@
+/server/v22.6/server/streams/metadata-and-reserved-names.html     /server/v22.10/streams.html#metadata-and-reserved-names   301!
+/server/v22.6/server/threading.html                                  /server/v22.10/server-settings.html#threading   301!
+
 ## Server version 21
 /server/generated/21.2/docs/*               /server/v21.10/:splat                      301!
 /server/v21.2/*                             /server/v21.10/:splat                      301!
 /server/v21.6/*                             /server/v21.10/:splat                      301!
 
+/server/v21.10/indexes/configuration.html                        /server/v21.10/indexes.html#configuration-options   301!
+/server/v21.10/projections/system-projections.html                /server/v21.10/projections.html   301!
+/server/v21.10/projections/configuration.html                     /server/v21.10/projections.html#configuring-projections   301!
+/server/v21.10/indexes/tuning.html                               /server/v21.10/indexes.html#tuning-indexes   301!
+/server/v21.10/networking/heartbeat.html                         /server/v21.10/networking.html#heartbeat-timeouts   301!
+/server/v21.10/networking/endpoints.html                          /server/v21.10/networking.html#exposing-endpoints   301!
+/server/v21.10/installation/compatibility.html                   /server/v21.10/installation.html#compatibility-notes   301!
+/server/v21.10/security/authentication.html                      /server/v21.10/security.html  301!
+/server/v21.10/clustering/node-roles.html                         /server/v21.10/cluster.html#node-roles   301!
+/server/v21.10/clustering/using-ip-addresses.html                 /server/v21.10/cluster.html#cluster-with-gossip-seeds   301!
+/server/v21.10/diagnostics/logging.html                            /server/v21.10/diagnostics.html#logging   301!
+/server/v21.10/security/authentication.html                      /server/v21.10/security.html#authentication   301!
+/server/v21.10/networking/tcp-configuration.html                 /server/v21.10/networking.html#tcp-configuration   301!
+/server/v21.10/projections/user-defined-projections.html           /server/v21.10/projections.html#user-defined-projections   301!
+/server/v21.10/clustering/acknowledgements.html                   /server/v21.10/cluster.html   301!
+/server/v21.10/security/trusted-intermediary.html                 /server/v21.10/security.html#trusted-intermediary   301!
+/server/v21.10/indexes/advanced.html                             /server/v21.10/indexes.html   301!
+/server/v21.10/diagnostics/vector.html                            /server/v21.10/diagnostics.html#vector   301!
+/server/v21.10/introduction/clients.html                          /server/v21.10/#protocols-clients-and-sdks   301!
+/server/v21.10/persistent-subscriptions.html                       /server/v21.10/persistent-subscriptions.html   301!
```

Comment:
This doesn't seem right? It looks like this is setting up a permanent redirect to the same page?
Maybe this was originally from 21.6 to 21.10.

Reply:
@cdevarenne There are two main reasons why these links were broken:

1. unsupported versions 
2. change of URL structure (the case for a lot of these): This restructuring has altered the URLs, and even though the content hasn't been removed, its address has changed. This discrepancy between the expected and actual URL paths leads users to broken links.

So for e.g. /server/v21.10/indexes/configuration.html needs to redirect to server/v21.10/indexes.html#configuration-options since configuration.html no longer exists in that path.

Reply:
The github comment box ends up showing more than the line I was commenting on. So to be sure I am asking about

```
/server/v21.10/persistent-subscriptions.html                       /server/v21.10/persistent-subscriptions.html   301!
```